backend/src/ApplicationCore/Entities/b2c/Token.cs

```diff
@@ -0,0 +1,43 @@
+﻿using PartyKlinest.ApplicationCore.Entities.Orders;
+
+namespace PartyKlinest.ApplicationCore.Entities
+{
+    /// <summary>
+    /// fields that are included in the Azure AD B2C claims that will be necessary to process requests
+    /// </summary>
+    public class Token
+    {
+        /// <summary>
+        /// unique GUID, used to distinct registered users
+        /// </summary>
+        public string OID { get; set; }
+        public UserType UserType { get; set; }
+        public string Email { get; set; }
+        public string Name { get; set; }
+        public string Surname { get; set; }
+        public Address Address { get; set; }
+        public bool IsBanned { get; set; }
+
+        public Token(Claim claim)
+        {
+            OID = claim.ObjectID;
+            UserType = b2c.UserTypeConverter.StringToEnum(claim.AccountType);
```

Comment:
Raczej nie ten konwerter, pole extension_AccountType w claimach przyjmuje wartości: Client: "0", Cleaner: "1", Admin: "2" (nie są to inty, tylko stringi)

Reply:
Ogólnie trochę głupio wyszło, ale właśnie w API są te ładne tekstowe wartości enuma (np. "Cleaner"), a Azure (przynajmniej przez Reacta) zwraca "0", "1" lub "2"